include/SKR_mini_E3-2.0/Pins.h

```diff
@@ -54,7 +54,11 @@
 #define E_ENABLE_PIN        PD2
 #define E_END_PIN           PC15    // E0-STOP
 
+#if defined(USE_LEONERD_DISPLAY)
```

Comment:
This provides inconsistencies to the other Pins.h files. 
Since this definition is irrelevant, please remove.

Reply:
This was done to free up resources